newtests/lsp/completion/test.js

```diff
@@ -130,9 +130,9 @@ export default suite(
                 },
                 {
                   label: 'aUnion',
-                  kind: 6,
-                  detail: 'number',
-                  inlineDetail: 'number',
+                  kind: 13,
```

Comment:
this is unrelated, but the test runner doesn't catch this. it is doing a very fuzzy string match and `kind: 6` and `detail: 'number'` exist elsewhere in the response; and it doesn't check that the fragments in the actual response exist in the expected response.